src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
Then create a new PR on which both wet-boew/GCWeb#1237 and #7983 would be dependent